### PR TITLE
Vaibhav1

### PR DESCRIPTION
**Pull request by vaibhavj18**

### CODE REVIEW

**Review by yashubari — APPROVED** [review]

File: Assignments/HTML_CSS/Assign1/vaibhav/vpE1.html [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+            <label class="label-details" for="time">Time:</label>
+
+            <input type="time" id="time" name="time">
+
+.content {
+    max-width: 500px;
+    margin: auto;
+    background: white;
+    padding: 10px;
+
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign1/vaibhav/vpE1.html:68]: what is this

File: Assignments/HTML_CSS/Assign1/vaibhav/vpE1.html [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    </head>
+
+</html>
+
+
+<html>
+
+    <head>
+
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign1/vaibhav/vpE1.html:205]: what is this can you explain

File: Evaluation/Eval_28Nov20/vaibhave1/vpE1.html [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+    <head>
+
+        <title>vaibhav dry fruits</title>
+
+        <link rel="stylesheet" href="vaibhav.css">
+
+    </head>
+
+    <body> 
+    <body text ="teal" bgcolor ="BLACK">
```
Comment (Pawan1005) [inline Evaluation/Eval_28Nov20/vaibhave1/vpE1.html:14]: why 2 body tags

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
check before push 
need to change every thing 
add indentation 
learn where we put css

**Pull request merged**